pkg/api/dtos/dashboard.go

```diff
@@ -31,6 +31,7 @@ type DashboardMeta struct {
 	FolderUrl             string    `json:"folderUrl"`
 	Provisioned           bool      `json:"provisioned"`
 	ProvisionedExternalId string    `json:"provisionedExternalId"`
+	Provenance            string    `json:"provenance"`
```

Comment:
shouldn't provenance be boolean to indicate the dashboard is immutable or not?

Reply:
we may want to differentiate this behaviour for specific sources therefore we have decided in favour of a string property.